main.go

```diff
@@ -134,8 +144,13 @@ func cmd() *cobra.Command {
 		},
 	}
 
-	cmd.PersistentFlags().StringVar(&maxVersion, "provider-version", "",
```

Comment:
What was previously `provider-version` is now `max-version`, and the old description for `provider-version` more accurately describes the new `target-version`